test/integrationTests.test.ts

```diff
@@ -174,7 +184,7 @@ describe("Read requests", () => {
   });
 });
 
-describe("Write requests", () => {
+describe.skip("Write requests", () => {
```

Comment:
Do we want to continue to skip these tests?

Reply:
Yeah, I usually leave them skipped unless I'm specifically testing creation since they create new content in whatever space you have configured